### PR TITLE
Rust 2 a

### PR DESCRIPTION
**Pull request by coriolinus**
Expansion of Rust 1 a: handle arbitrary clocks.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -78,6 +78,14 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": []
+    },
+    {
```
Comment (ErikSchierboom) [inline config.json]: There is a merge conflict in this file.

File: exercises/rust-2-a/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+## Introduction
+
+This is Part 2 of our research into how people write Rust in varying styles. This exercise should take 10-15 minutes. There is no right or wrong way to approach it - just do what feels the most natural to you. Feel free to use books, Google or Stack Overflow, just like you would if you were programming normally, but please don't look for a complete solution to the problem as this will negatively affect the research.
+
+## Instructions
```
Comment (ErikSchierboom) [inline exercises/rust-2-a/README.md]: Maybe the instruction could be a little bit more explicit about the student being given some existing, working code? As an example, consider the go-2-a exercise's [instructions](https://github.com/exercism/research_experiment_1/blob/master/exercises/go-2-a/README.md#instructions).

File: exercises/rust-2-a/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+pub struct Hand {
```
Comment (ErikSchierboom) [inline exercises/rust-2-a/src/lib.rs]: The second part of the exercises are structured a bit differently. There should be multiple starting points, which are defined in the exercise's `.meta/start/<start number>` directory. For an example see the Go 2-a exercise's `start` directory: https://github.com/exercism/research_experiment_1/tree/master/exercises/go-2-a/.meta/start

The idea is that in the second part of the exercise, students get assigned one of the different starting points (which should use different approaches), and then expand/modify the code to fit the updated requirements. That way, we can see how the different starting points influence the outcome. Sorry that this wasn't clear enough.

**Comment by coriolinus** [comment]
@ErikSchierboom Changes made.

**Pull request merged**